packages/strapi-admin/admin/src/components/Inputs/Wrapper.js

```diff
@@ -9,7 +9,7 @@ import { colors, sizes } from '@buffetjs/styles';
 
 const Wrapper = styled.div`
   position: relative;
-  padding-bottom: ${sizes.margin * 2.3}px;
+  padding-bottom: ${sizes.margin * 2.7}px;
```

Comment:
This is indeed something we should implement correctly in one dedicated PR (in order to change every values and to use sizes to set the theme instead of using it directly)!

Reply:
Agree!